tests/arc/texceptions.nim

```diff
@@ -14,3 +14,13 @@ block: # issue #13071
       a = e.msg & $e.name # was segfaulting here for `nim cpp --gc:arc`
     doAssert a == "foo:MyExcept"
   fun()
+
+
+block: # issue #17351
```

Comment:
```suggestion
block: # bug #17351
```

https://nim-lang.github.io/Nim/contributing.html
> Always refer to a GitHub issue using the following exact syntax: bug #1234 as shown above, so that it's consistent and easier to search or for tooling. Some browser extensions [...]